### PR TITLE
50088 Improved Amsterdam Schema support - Source NAP Peilmerken from Amsterdam Schema

### PR DESCRIPTION
**Pull request by JKoetsier**

### CODE REVIEW

File: gobcore/model/amschema/model.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+NonObjectProperties = Union[StringProperty, NumberProperty, IntegerProperty, RefProperty, BooleanProperty]
+
+
+class ObjectProperty(Property):
+    type: Literal["object"]
+    properties: dict[str, NonObjectProperties]
+    relation: Optional[str]
+
+    def _is_relation(self):
+        return not not self.relation
```
Comment (roel-devries) [inline gobcore/model/amschema/model.py:101]: not not

File: gobcore/model/amschema/model.py [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+        }
+
+
+class ArrayProperty(Property):
+    type: Literal["array"]
+    items: Union[NonObjectProperties, ObjectProperty]
+    relation: Optional[str]
+
+    def _is_relation(self):
+        return not not self.relation
```
Comment (roel-devries) [inline gobcore/model/amschema/model.py:134]: not not

**Review by roel-devries — APPROVED** [review]
Looks good!

File: gobcore/model/amschema/model.py (hunk 1 above)
Comment (JKoetsier) [reply gobcore/model/amschema/model.py:101]: Boolean conversion :)

File: gobcore/model/amschema/model.py (hunk 2 above)
Comment (JKoetsier) [reply gobcore/model/amschema/model.py:134]: Boolean conversion :)

**Pull request merged**